streaming/base/dataset.py

```diff
@@ -265,6 +267,9 @@ class StreamingDataset(Array, IterableDataset):
             of this size, and samples within each block are shuffled. Defaults to ``1 << 18``.
         sampling_method (str): Which sampling method to use, either ``balanced`` or ``fixed``.
             Defaults to ``balanced``.
+        sampling_granularity (int): When picking samples for a stream's final partial repeat,
```

Comment:
Can you please add more details on what other value means apart from `1` for the users so that they know how to tweak this parameter based on their need?

Reply:
added another example